### DISCUSSION TITLE
part-ii/2018-2/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#183] [Question comments]
Part II 2018 Algebraic Geometry: Paper 2, Section II, I 

https://questions.tripos.org/part-ii/2018-2/

**Comment by silkca** [comment]
The first part is some bookwork and a simple application of it.

The second part is quite simple to manipulate, although it may not be correct - $U$ is certainly a quasiprojective variety, so it is closed in some open set, but it may not be closed in $W \times k^{3}$. Regardless, it is easy to split the definition of $U$ into closed and open conditions, which eventually determines how it fits in the topology on $W \times k^{3}$.

**Reply by artie2000** [reply]
What if $f-1$ is reducible? I'm not sure what to do..

**Comment by artie2000** [comment]
One way to fix this question is to pretend is says "$Z(f-f(p))$ is smooth at $p$".
